### PR TITLE
DbVisualizer PR for sql client docs

### PR DESCRIPTION
**Pull request by BloomBoy**
We want DbVisualizer to be added to your docs and have prepared the changes according to your instructions and best effort.

### CODE REVIEW

**Comment by rfraposa** [comment]
Please update your branch first, in order to resolve the conflicts with `sidebar.js`.

**Comment by BloomBoy** [comment]
Updated now.

**Comment by rfraposa** [comment]
@BloomBoy  - there are a lot of files being modified that seem out of scope for what you are trying to do. I don't know if your fork is old or some other issue, but this PR should only be editing the integrations page, the sidebar, and introducing your new pages and image. Thanks!

**Review by rfraposa — CHANGES_REQUESTED** [review]
Files like ssl-user-auth should not be modified in this PR

File: sidebars.js
[the remark this answers is not in the document]
Comment (rfraposa) [reply sidebars.js:1]: OK - I pushed a change to your fork to get your sidebar.js file up-to-date

**Review by rfraposa — APPROVED** [review]

**Pull request merged**